scandium-core/src/main/java/org/eclipse/californium/scandium/config/DtlsConnectorConfig.java

```diff
@@ -1108,8 +1109,11 @@ public Builder setIdentity(PrivateKey privateKey, Certificate[] certificateChain
 		public Builder setTrustStore(Certificate[] trustedCerts) {
```

Comment:
should we deprecate `setTrustStore(Certificate[] ...)` and create a new `setTrustStore(X509Certificate[] ...)` ?

Reply:
if  the certs are loaded from a "KeyStore", then they are `Certificate`. 
And, if we change it, we should change it also in `setIdentity`.

Reply:
Ok it sounds to be a good argument to keep it like this.